proto/near/get_public_key.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+import "error.proto";
+
+package near;
+/**
+ * Example of successful get_public_keys generation
+ *
+ *        Host                       Device
+ * GetPublicKeyIntiateRequest  =>
+ *                            <= GetPublicKeyResultResponse (1-10 public keys)
+ * GetPublicKeyFetchNextRequest =>
+ *                            <= GetPublicKeyResultResponse (11-20 public keys)
+ * GetPublicKeyFetchNextRequest =>
+ *                            <= GetPublicKeyResultResponse (21-30 public keys)
+ */
+
+
+/**
+ * Example of successful get_public_key generation
+ *
+ *        Host                       Device
+ * GetPublicKeyIntiateRequest  =>
+ *                            <= GetPublicKeyResultResponse (1 public key to be sent after user confirmation)
+ */
+
+enum GetPublicKeyStatus {
```

Comment:
Rename the prefix for all enums and message to GetPublicKeys

Reply:
Updated